src/mk.sh

```diff
@@ -39,8 +39,8 @@ function mk_build
 
   PARALLEL_CORES=$(( $PARALLEL_CORES * 2 ))
 
-  say "make -j$PARALLEL_CORES $MAKE_OPTS"
-  make -j$PARALLEL_CORES $MAKE_OPTS
+  say "make -j$PARALLEL_CORES ARCH=${configurations[arch]}"
+  make -j$PARALLEL_CORES ARCH=${configurations[arch]}
```

Comment:
A reeealy nitpick: when using `${configurations}` for bash commands, it's better if we protect it with single quotes so that strings with spaces get correctly parsed by the command. In this case, this is veeeeery unlikely as we don't expect `${configurations[arch]}` to be a string with spaces. But just to make sure, I think we could write this as `... ARCH='${configurations[arch]}'`.